src/js.c

```diff
@@ -1538,6 +1538,7 @@ jsSub_processSequenceMismatch(natsSubscription *sub, natsMsg *msg, bool *sm)
     jsSub       *jsi   = sub->jsi;
     const char  *str   = NULL;
     int64_t     val    = 0;
+    struct mismatch *m = &jsi->mismatch;
```

Comment:
Can m be null?

Reply:
No, this is a pointer to an internal structure (take the address of). I could have name those fields say "mldseq", etc.. but decided to make them part of a struct.